src/overlays/actors/ovl_En_Takaraya/z_en_takaraya.c

```diff
@@ -5,6 +5,7 @@
  */
 
 #include "z_en_takaraya.h"
+#include "objects/object_bg/object_bg.h"
```

Comment:
Could you move this into the header file? It's needed in there for `OBJECT_BG_1_LIMB_MAX`